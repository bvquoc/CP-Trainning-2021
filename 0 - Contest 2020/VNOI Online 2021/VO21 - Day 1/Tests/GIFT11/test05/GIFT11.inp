10
200
45893 148105012 343 902230217 651 215 7854492 8385143 164730832 36694 596 2 52986 8092074 306 77165 221 242453996 5308837 43767 337168051 3 32288 508800940 2 432345612 159912290 7809238 59179 640501341 334 6 9399342 908 6025571 689 1 8 260 7650406 9283041 85599 78784 18410 312502611 8 5312626 975521584 699 8314272 41042 286066752 42213 9547987 4 3 556633846 565593405 367971663 4682803 629 7 471 682243080 52312 942866333 6555440 1124667 402513962 10469 473 9548133 264 1 69070 4989792 69566 444310496 486158251 315648707 8 9380934 351 622 303901336 80563 8146626 88587 595909177 919017887 545982077 64449 9278223 137 120 920 627 3252763 658950462 233982861 5652221 889461469 2956759 801029237 363 43667 860 3 7225705 61477 6 53336 96269 48701 637844398 3 104164315 627500612 377 203491012 3103453 770950706 593182699 4968416 98699 271566114 955 873 14911 655 9443021 24720 41137 155 991979138 1872343 77515 341772910 1 8396958 5620774 799 1294241 714 29065 6 31837 3917552 688 77622 276 7062958 603454242 9125046 7658971 6925168 50072 4548848 652571503 40159 4 998799723 91053 87173 970 767080033 694 8414537 804 987 376 41110 460592259 2968865 5 621 3391143 6 8880910 6 361 3 2254917 6 1715975 877872875 332 60369 255503702 3368047 3522668 955813086 914 974322108 418 290 8374781 8298764 572416732 58866
200
755 713515736 1 90352 2716353 815031321 4 524 72005 694315271 512327231 8 63820 68636 463 1884853 791473038 20545 895838910 771 233 946445791 96362 2021951 424457785 531 99888 680 42164 664281001 7852746 41265 357767358 215 59167 830 457 3045442 161063385 238830885 112498277 418 922 774 5 2218993 393429025 1485842 938 92632 849 4 537034901 8549632 770796329 44128 47577 52306 2089812 1981224 9712268 8733558 3 543355399 474 94278 73810 34246 25249 294153435 165 182221087 913 6372270 461 998 941655388 8 435772305 3 6157759 55920 7790308 29994 429672006 890 933 27842 636752346 520770774 4 624 566335397 493 5550121 54845 975 759 9190775 827 84626 7121126 3756825 1 403 851 274 54922 21734 4580846 374 1633535 295273384 1 78441 365140991 902294359 210 850 9341573 37996 9829291 94925 842873577 9632351 52404 94010 908551557 4348170 9723832 1 41280 505 1013537 995 945 6200505 260 336 6 280 5442710 45363 375661391 680969203 28467 27839 4647285 962 3 7543212 44777 2 2002098 7718099 75012 5211450 11540 80460 769 537672293 563965371 74498 507 480 64686 773 558 50573 310298743 8022854 406146335 424 667 118137318 91296 227 870489006 25894 41148 190 350089967 9508591 76479 7381379 985 23834 61439 51238 231113046 848 876184903 565 458 1 5 4941100 882408876 869 562
200
981 971705351 51630 580954419 830378953 9 91274 706 140 7229242 8221758 8 771 3812229 950542451 7135461 3 5752785 8258498 152360747 535 4 5401460 601 672987390 931 6455250 115 67250 83178 801 8350142 14023 6 22742 4585368 87110 920 542 57109 10393 416 306 414007230 9345321 212103948 326658360 6666812 866 322 175 98921 4475025 6911756 539708777 9 705 2374440 6343211 623477585 100 2064122 334080426 138024670 1 9890489 63184 9 3008389 76640 856 63065 395953256 282 68448 2013304 968 2 421673342 767123492 46277 489 513 863 91905 552 5 54969 732 9939567 214 7 6 947033579 64801 6453071 96309 448802780 136439978 634157908 12405 356 3 905 9179480 9 7 9896259 656 965 1805828 425927453 15645 128 84816 35558 2 3264261 4 920008854 3446108 863854438 39153 3277223 292 2374476 4166520 456 629 48128 438 632 141 52794 486 5808851 101 54453 31495 755840223 300 405616136 6 89448 10198 888 112 2 319103506 2119244 523 48469 6864125 24735 5992381 4855147 974643825 2 8297921 35171 29367 1408181 218909794 533 546849239 496673289 8961899 24746 490 1261142 824256164 555054732 71882 33363 229 939 82800 133 597 588 3272724 923 532 7981220 867 83480 2450138 208501953 256 588859171 638 707538183 69329 71786 829037664 693089487 8 401 824743205 915707708
200
859353837 312 897 506 220679173 1980367 29407 371 8534203 152 697190055 8302394 239 236730731 473587586 944815749 155 204589747 21193 2 2254631 3 5 293566523 281 7536657 661298206 160546202 941 10714 1714399 711051355 2999905 9 392 829 772030177 54284 700585834 54620 52759 643627975 686 522 9988667 294019269 64877 886986998 17351 134637102 7 395 915 677 14592 3625921 6 557 8670174 626 5416146 733030124 28139 2 73605 187 8098784 36697 306 2 5026166 1899194 395939956 9 8656614 12498 52784 60511 1 857 976 414840027 728 25624 882730977 184 2801580 667803480 9 9141042 575472863 482 97911 902912879 2394520 4381280 55595 968 89729 678861418 196445372 393 725600899 519414490 1003394 49993 5 92535 6408294 25947 742128950 14700 827125650 629 9695514 355210561 159 3 801180154 336 577 146755500 567996384 66384 396548830 9863779 40261 776444988 6664130 946 54467 156 355 6 6759761 6467530 169 961355755 8 746 2 6702583 509 35681 590949306 17671 2 62044 1 8877716 930763848 7477304 7743141 4112726 123 71055 577 9709320 886508436 425 7961530 193 685 5 126 63227 46288 8 1818561 1831918 4430668 677 87542 659762068 75428 270 861326594 204786183 791680863 391 181283119 20653 160 915395522 57477 770 538 547 59534 3599390 320 899 6380946 437 9 9 315 137475261 976 488
200
4 8 428 101 710433983 427 26436 9 119 33095 80548 944522548 404 524 98280 95661 67815 15583 741205611 202203776 458954760 428 5648660 48422 776 7498242 202 61273 17869 410181584 9391489 536 1399300 1 657 779 649 72978 7590709 3228164 885 968024857 7 9534034 397159922 99940 670 7218760 811137193 497 4489328 38834 253645954 1836700 5321942 6597718 2 5 283768308 73999 4835712 2 87363 25109 36839 8568606 88315 3 552 41043 606313591 703722988 87473 4647851 886452495 5372544 28054 660 751106956 421855045 401433389 1433259 598070839 66212 2817054 2736069 2489305 72855 84163 73655 90219 410 2 2 748164070 71242 206633035 76757 7 236 672072384 2390772 38586 162222566 7990189 680170223 573243510 721307648 518892602 584718734 343780191 74547 205344849 6998667 390186942 293327224 2 8 21724 565 44399 716861891 382 4801694 713753513 7961647 305 810979664 9546240 8 379 1879545 4324077 286 8361407 361257200 8 282329635 3629727 276 753823216 3650157 1321012 7 3 92684 5 52925 676 66986 179304609 156 4694653 68400 68011 207450967 87087 645336710 2006030 364198967 7292630 6 336 820554006 40635 4598090 8 625523468 352 589 8 85682 556908806 572 348552329 144310753 893 53166 259476496 139 123 7162406 848854823 3387840 3945244 9 51093 5018721 846601813 39583 980 675104175 8 225 6502878 821550340 1 362 255323274 485396283
200
996 675 5253042 5100260 9441597 7264125 529801184 6049762 781 736887056 703072731 4208778 3433814 7790291 46221 7470773 544 261 54224 7 7 67048 948582971 68041 8489893 25820 5 108734660 113949626 97316 9927569 948617078 570261736 37815 6 648 3686998 937979823 25351 5858612 7685357 140 7 126692658 3547671 512878385 172642357 204843643 31068 23085 154570220 7 854 84834 7 7938259 358212510 15145 185 188 67964 918 921 764 9485593 7952671 9 65853 8846415 34945 712347664 11174 19735 5 7 755962730 5602489 31648 981953214 6233426 735660094 258597742 1858168 8585125 4775408 2359565 7 710 836 6643606 54370 1 966 458455681 670987618 328 56251 645 19111 8827298 1027945 4 5 2201098 2994977 6662927 393 5162996 771 452389055 632610631 966422970 74465 31808 99782 949580915 670897979 5 770479046 194 1 47140 5 127 443677233 278746618 233 9061844 3948767 799126115 9 8 727629605 910 1643075 4 8 733218246 115 9 539371871 3 7948690 7145651 63173 198443939 5012501 8103570 89431 126 149 88295 2997519 359 9143551 890158315 433 8489763 55320 959664149 691 5 16053 43760 480974721 81983 19807 6 5426201 305512429 8048799 3880123 727118755 76850 802 7718397 4015305 5 769 8725698 79204 51204 2854845 5211750 4229260 52324 551 575 7194938 7045246 100 4497192 398143592 702785963 4907269 33593 922 3051974 39643 8800587
200
33606 242 691 956 719487177 81925 7 9888663 10319 2179164 135368374 756 8671650 2251876 4166618 989 8173418 51013 184 2731876 480 83977 3752721 415 878 56201 220630134 2667426 3 681 3 9900931 6269973 5 3347201 5293254 6 3004841 78008 661958403 1632630 358057442 4685127 8876630 452 77476 628 997 29561 943 854 939906523 9 2577733 203 6644736 3150709 682894007 60041 3713693 85004 974 359 12396 1558884 511783563 267 1248740 376300983 642 1 4 623113085 465 9627866 709 372 909 695 35558 28041 7 1980357 626 4 7795699 751 2980997 642 2446791 272 245 82381 797019680 577515926 94805 2264533 6185974 7549735 357161172 820165504 599789316 49846 78862 114666406 91201 72990 321 3289905 131902183 883167351 633 526 715 44121 830 2703576 7 12127 84476 427 41287 946470352 489 3647806 64842 734499319 448733456 986 3390291 69964 141 943203881 929602139 487 488537773 892 423723994 85043 3662258 3 455015690 98407 6 4 501728164 886 2 19187 121 751625680 118768003 345 5688505 465691078 215 771639968 6 582722264 80684 3 2993313 838200401 16967 912 583 92089 5640221 263 2614052 997 679358792 19862 34270 6 994907776 689812078 74906 15604 4358046 712009582 3964129 6993634 373231484 4 8479451 4582716 8 428 29379 6 412 603964140 4781280 901019176 968327818 8100448 643 8496076 988
200
42166 6973407 6963823 8468911 90298 915 488 5515669 720412434 657 5 447235177 8048010 4 109 5641029 545 3388847 84627 75601 557936392 117066608 967054878 161016182 5532577 6110351 983 922 492 9 979 136 7529427 3 731 323 1225151 4792089 676 223 27703 9 4 650 345944967 16709 92217 580 376340406 8282227 3 903935394 225453517 5603679 78403 96757 610 6272880 4452888 789984706 16666 730031648 89476 64531 4797217 813996584 380 180 516802356 124877569 769 823698355 4215549 871 23996 435 14533 7610696 2 309925323 524 1981801 3 191937939 9554740 817225407 808 6 48061 378 31201 4113432 3 83900 29914 48551 8 5 64954 310 99451 921 54043 7 52468 449 485736710 2 900 144 993 7458183 90611 359 96671 730 56779 605017615 378 954570229 1940612 593 232950540 98863 9377624 590845225 63312 7364556 2891167 6 777 411199733 74915 51266 3 20116 129102316 9 5881473 16465 6791172 4241282 6027404 974781013 48084 1540263 38346 253202025 8115683 805 802 995 852734269 4 8 859 435883881 80471 2 902 3405584 830850341 16560 6284593 3062364 659834204 2233434 800070654 63317 275971189 2740862 697 452127301 837068648 5953956 5 1 85520 40439 220333134 613 61236 71454 130 48934 525 465501956 636 28874 378503926 1 889 50674 89302 8 5606847 831970006 724326407 834502019 573091645
200
2812255 679481650 219 7818770 228973607 9105280 438970493 979 8798935 773 5 47603 819 87879 5917064 3 74961 86435 9727777 223216921 211319213 67062 426 629 72565 340 5048043 1 2257521 927070243 69565 2 30682 4798112 5303993 9 803 1208456 111 76323 20586 744 383384786 250 8 8727484 4 522 360991788 233802810 3562240 75364 5659809 247 257 325692810 87560 603221979 186308938 401966284 508620653 9 65000 2159971 4297750 4 9394808 1 59295 13617 83955 79189 9 319 4520210 186 15631 298399297 94382 768467144 117 6 304 490382272 933925749 381 4854753 128 497 68399 813138218 3726284 152013513 315 55412 413 2 3838544 860 417190412 356 292 4282447 595 9367684 500 707063058 834 812 4844088 81162 610 3561862 9587738 956 393 369864875 5277083 265387581 5155855 2 51460 314 99981 27751 20622 2 82448 479588113 726912273 9343673 5 366 9771399 67692 1853259 506715242 49225 30547 9917444 38155 812 3779127 377340742 89685 73716 6829410 799 57233 5 1375451 3132427 549960041 8388106 820 459 8595942 4873790 8 63232 232 761910851 1 672 921 10372 350 143 361 7 770368597 446 22730 840 119474726 9246367 6152996 63679 51764 874 9202361 47972 129479034 4 328531585 6 511 2 206 667 849 374 4079568 9106407 706762878 279 6 525 67897 186
200
901 506776839 833 3244522 473442890 462347970 604016661 417 163409061 8911483 6650856 84681 972987538 11322 53865 3517622 11786 9 483 7399734 547991047 605 31613 61748 4146657 32667 229561794 612 518 75103 3995785 223685899 6 444 39229 858 275856796 633 7017926 1787395 883100166 4194279 1544268 122 3840203 894768337 13306 7 3876948 2293362 1 36139 169 897712452 55582 663 8408787 808 959 720996801 3457692 383 42347 150707202 9 990689369 94306 68196 184486478 156 5028215 6459727 409984009 38277 994 312 974 62073 595967324 817 819813774 216 945 149 4 9898536 2428384 8429899 266 2083062 122766989 2846091 61447 190212015 660865741 162818571 8325912 552 5187369 871 243853665 929 550930154 111257580 35724 956 374916888 641373887 5 274 6 4920246 4020330 415 8 64616 5686511 582 7578643 844 93946 15838 89492 927 7116564 82993 3041328 7880676 804 59951 1 546736130 7745430 903 7672797 59657 28817 920 111880640 58419 214376126 6060076 502 2316887 572657383 84339 26380 144828656 5 953480324 7 5 226817583 122599158 358469980 375 772044257 86179 2 65919 337221650 232 331784504 775 305993387 37793 5098513 50744 970393558 813 2657384 52712 8 4914135 731 43402 78533 80768 75781 598209673 93770 2 824 9989422 2 7219263 33093 4500688 127 5016902 8770617 462 152 341262665 1 4452340 61121 9 523 351
