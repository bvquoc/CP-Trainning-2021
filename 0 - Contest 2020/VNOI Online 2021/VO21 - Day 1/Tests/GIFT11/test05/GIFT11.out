684094614
965415784
726520888
634766739
122605179
394889124
912979050
496715962
145059480
337267290
